frontend/src/components/Footer/footer.module.scss

```diff
@@ -703,7 +702,7 @@ a > span {
   }
 }
 
-@media screen and (max-width: 510px) {
```

Comment:
this change shouldn't have come here, make sure to pull the latest master, rest looks fine

Reply:
Sorry for that.

Reply:
Its okay, we are all learning 😊